.gitignore

```diff
@@ -2,6 +2,8 @@
 *.pico
 *.dSYM
 *.exe
+*.dll
+*.obj
```

Comment:
_.gitignore_ already has appropriate patterns for `.dll`, no?  To date, Windows support has been aiming at Cygwin and GCC so `*.o` has sufficed, but it would be good to put `.obj` alongside `.o`/`.pico`.